src/CommonLib/ThirdParty/PSOpenAD/SSPI.cs

```diff
@@ -0,0 +1,676 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+
+namespace SharpHoundCommonLib.ThirdParty.PSOpenAD;
+
+internal static partial class Helpers
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct SEC_CHANNEL_BINDINGS
```

Comment:
Pretty sure these are the names from the shared library, but we should follow our managed code's styling